src/document/json/rht.rs

```diff
@@ -0,0 +1,307 @@
+use crate::document::time::ticket;
+use std::cell::RefCell;
+use std::collections::HashMap;
+use std::rc::Rc;
+
+/// RHTNode is a node of RHT(Replicated Hashtable).
+struct RHTNode {
+    key: String,
+    val: String,
+    updated_at: ticket::Ticket,
```

Comment:
I have a simple question. Is there any reason to use `ticket::Ticket` as opposed to other objects like HashMap or RefCell?

```suggestion
    updated_at: Ticket,
```

Reply:
At first, I tried to explicitly mark the package because the parent package was different. But after seeing your comment and looking at the code again, it seems that there is no need for that, so I modified it :)